test/SeqFocus.lean

```diff
@@ -0,0 +1,19 @@
+import Mathlib.Tactic.SeqFocus
+
+example : (True ∧ (∃ x : Nat, x = x)) ∧ True := by
+  constructor
+  constructor
+  map_tacs [trivial, exact ⟨0, rfl⟩, trivial]
+
+example : ((True ∧ True) ∧ (∃ x : Nat, x = x)) ∧ (True ∧ (∃ x : Nat, x = x)) := by
+  constructor
+  constructor
+  map_tacs [(constructor; trivial), exact ⟨0, rfl⟩, constructor]
```

Comment:
The fail tests seem to have been removed? They can be added using `fail_if_success` like in my earlier suggestion

Reply:
My apologies! I did not notice the changes. I'll restore the tests as you suggest.